client/src/components/Tea/TeaEditor/TeaEditor.jsx

```diff
@@ -3,123 +3,125 @@ import React from "react";
 import { Link } from "react-router-dom";
 import DataList from "../../DataList";
 
-export const TeaEditor = props => {
-  return (
-    <div className="container">
-      {props.flash.name ? (
-        <div className="notification is-success">
-          {props.flash.name} has been succesfully saved.{" "}
-          <Link to={"../../tea/" + props.flash.teaID}>View details</Link>
-        </div>
-      ) : (
-        ""
-      )}
+export class TeaEditor extends React.Component {
```

Comment:
yes! this is something i've been considering. i think it would also make testing a little more targeted. and it will become more of an issue as this form grows.